components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -168,14 +181,27 @@ export default function ProjectSettingsView() {
                 />
                 {enablePrebuilds && (
                     <>
-                        <InputField label="Workspace machine type" disabled={!enablePrebuilds}>
-                            <div className="max-w-md">
-                                <SelectWorkspaceClassComponent
-                                    disabled={!enablePrebuilds}
-                                    selectedWorkspaceClass={project.settings?.workspaceClasses?.prebuild}
-                                    onSelectionChange={setWorkspaceClassForPrebuild}
-                                />
-                            </div>
+                        <SelectInputField
+                            disabled={!enablePrebuilds}
+                            label="Build branches"
+                            value={prebuildBranchStrategy}
+                            containerClassName="max-w-md ml-6 text-sm"
```

Comment:
**issue**: For some reason, I don't see these changes reflected in the preview environment. The select element is still not intended under the checkbox element, maybe the preview environment is outdated?

<img width="572" alt="Screenshot 2023-09-18 at 13 34 53" src="https://github.com/gitpod-io/gitpod/assets/120486/4befcdf6-8b5d-4f75-a14e-78d2eddc1b1c">

Reply:
Sorry for that. I fear it was a race with terminating `telepresence` on that preview env. It's synced now.